### PR TITLE
[10-10CG] Remove caregiver1010 flipper logic for SubmissionJob

### PR DESCRIPTION
**Pull request by coope93**
## Summary

- This work is behind a feature toggle (flipper): NO
- Removing the flipper toggle logic for the SubmissionJob because it is turned on in production now. The change was introduced in [this PR](https://github.com/department-of-veterans-affairs/vets-api/pull/18814).
- 10-10 Health Apps

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/93615

## Testing done

- [x] *New code is covered by unit tests*

## What areas of the site does it impact?
- 10-10CG

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

**Comment by stevenjcumming** [comment]
@coope93 please have a teammate review, then I'll review

File: app/sidekiq/form1010cg/submission_job.rb [hunk 1]
```diff
@@ -16,17 +16,10 @@ class SubmissionJob
     end
 
     def retry_limits_for_notification
-      return [1, 10] if Flipper.enabled?(:caregiver1010)
-
-      [10]
+      [1, 10]
     end
```
Comment (Mitch-A6) [inline app/sidekiq/form1010cg/submission_job.rb:20]: 🤔 now that this is reduced to a fixed constant, would it make sense to make it into one?
```suggestion
    RETRY_LIMITS_FOR_NOTIFICATION = [1, 10]
```

Comment (Mitch-A6) [reply app/sidekiq/form1010cg/submission_job.rb:20]: wait, never mind, this is part of the signature.

**Review by Mitch-A6 — APPROVED** [review]
🔥

**Review by stevenjcumming — APPROVED** [review]

**Pull request merged**